src/webui/www/private/scripts/misc.js

```diff
@@ -275,6 +276,36 @@ window.qBittorrent.Misc ??= (() => {
         });
     };
 
+    const downloadFile = (url, defaultFileName, errorMessage = undefined) => {
+        const xhr = new XMLHttpRequest();
```

Comment:
Would you consider using [fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) instead?

Reply:
Definitely. Will switch over.